<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="headerLayout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="header2Layout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="bodyLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="bottomLayout"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="bottom2Layout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
